plugin/documents.py

```diff
@@ -322,8 +317,6 @@ def on_text_changed_async(self, change_count: int, changes: Iterable[sublime.Tex
         if userprefs().document_highlight_style:
             self._when_selection_remains_stable_async(self._do_highlights_async, current_region,
                                                       after_ms=self.highlights_debounce_time)
-        self._when_selection_remains_stable_async(self._do_color_boxes_async, current_region,
-                                                  after_ms=self.color_boxes_debounce_time)
         self.do_signature_help_async(manual=False)
         self._when_selection_remains_stable_async(self._do_code_lenses_async, current_region,
```

Comment:
The same fix can be applied on `_do_code_lenses_async` but that can be done in a separate PR.